riptide-failsafe/src/test/java/org/zalando/riptide/failsafe/FailsafePluginRetriesTest.java

```diff
@@ -100,7 +100,7 @@ public RequestExecution aroundNetwork(final RequestExecution execution) {
                                     .withMaxRetries(4))
                             .withPredicate(alwaysTrue()))
                     .withPolicy(new CircuitBreaker<ClientHttpResponse>()
-                            .withFailureThreshold(3, 10)
+                            .withFailureThreshold(5, 10)
```

Comment:
@whiskeysierra honestly saying I'm not sure how this was working before. On L94 we configure max 4 retries, but after 3 breaker should open. So tests retriesConnectionFaultForNonIdempotentMethod and shouldRetryUnsuccessfully is supposed to fail (what actually happened with failsafe 2.4.0), but on 2.3.5 they was passing successfully.